src/app/(nextjs_migration)/(authenticated)/user/breaches/page.tsx

```diff
@@ -161,6 +161,45 @@ export default async function UserBreaches() {
     return breachRowsHTML;
   }
 
+  // Fetch list of subscriptions.
+  const bearerToken = session?.user.subscriber?.fxa_access_token;
+  if (!bearerToken) {
+    console.error("User has no bearer token");
+  }
+  const result = await fetch(
+    `${process.env.OAUTH_API_URI}/oauth/mozilla-subscriptions/customer/billing-and-subscriptions`,
+    {
+      headers: {
+        "Content-Type": "application/json",
+        authorization: `Bearer ${bearerToken}`,
```

Comment:
If this receives `Bearer undefined` will it always fail the fetch request? And if so, should we skip the network request entirely?